docroot/themes/contrib/civic/civic-library/components/03-organisms/large-filter/large-filter.js

```diff
@@ -268,10 +268,10 @@ CivicLargeFilter.prototype.renderHTMLFilterItem = function (key, label, type, th
   if (type !== 'input_radio') {
     return `
     <li class="civic-large-filter__tag">
-      <button class="civic-button civic-theme-${theme} civic-button--chip civic-button--button civic-button--extra-small" data-component-name="button" data-id="${key}">
+      <button class="civic-button civic-theme-${theme} civic-button--chip civic-button--button civic-button--small civic-button--dismiss" data-component-name="filter-chip" data-id="${key}">
```

Comment:
Fix issue on why selected filters are not styled properly in civic.

Reply:
please remove `filter-chip`